ldap/servers/slapd/daemon.c

```diff
@@ -2561,6 +2563,24 @@ configure_pr_socket(PRFileDesc **pr_socket, int secure, int local)
         }
     } /* else (!enable_nagle) */
 
+    if (!local) {
+#ifdef TCP_LINGER2
```

Comment:
I don't think we need the TCP_ defs here.

Reply:
Dear @Firstyear,

Thank you for your comment.
I have removed TCP_ defs .